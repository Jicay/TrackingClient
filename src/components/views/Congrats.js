import React, {Component} from "react";
import {Container,} from 'reactstrap';
import {Link} from "react-router-dom";

export default class Congrats extends Component {
    constructor(props) {
        super(props);
    }

    componentDidMount() {
        const {onLoad} = this.props;
        onLoad();
    }

    render() {
        return (
            <Container className="congrats">
                <div>
                    Bravo pour avoir soumis !
                </div>
                <Link to="/">Retour Ã  l'accueil</Link>
            </Container>
        );
    }
}