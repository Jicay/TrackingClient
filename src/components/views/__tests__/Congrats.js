import React from 'react';
import renderer from 'react-test-renderer';
import Congrats from "../Congrats";
import {MemoryRouter} from "react-router";

it('renders correctly', () => {
    const tree = renderer
        .create(<MemoryRouter><Congrats onLoad={jest.fn()}/></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});