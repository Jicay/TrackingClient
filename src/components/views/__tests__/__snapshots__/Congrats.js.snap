// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="congrats container"
  id="page"
>
  <div>
    Bravo pour avoir soumis !
  </div>
  <a
    href="/"
    onClick={[Function]}
  >
    Retour Ã  l'accueil
  </a>
</div>
`;
